example/StackNavigator.tsx

```diff
@@ -151,25 +151,38 @@ const StackRouter: Router<CommonAction | Action> = {
         });
 
       case 'NAVIGATE':
-        if (state.routeNames.includes(action.payload.name)) {
+        if (
+          action.payload.key ||
```

Comment:
Maybe we should also check if the `key` exists in the stack

Reply:
we do it line 180